tests/acceptance/features/cliCommands/cleanupEmptyTrashBin.feature

```diff
@@ -0,0 +1,14 @@
+@env-config
+Feature: delete empty trash bin folder via CLI command
+
+
+  Scenario: delete empty trashbin
+    Given the user "Admin" has created a new user with the following attributes:
+      | userName    | Alice        |
+      | displayName | Alice Hansen |
+      | password    | %alt1%       |
+    And user "Alice" has created folder "/folderMain"
+    And user "Alice" has deleted folder "/folderMain"
```

Comment:
we could also add these steps here
- delete file/folder
- restore from trash
- delete from trash